include/deal.II/meshworker/simple.h

```diff
@@ -107,11 +107,12 @@ namespace MeshWorker
       template<class DOFINFO>
       void assemble(const DOFINFO &info1,
                     const DOFINFO &info2);
-    private:
+    protected:
```

Comment:
Can you comment on why it is necessary to make all of these member variables `protected`?